packages/mdc-toolbar/index.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+export {MDCToolbar as default} from './component';
```

Comment:
should we have a default? Is this for backwards compatibility?

Reply:
Not sure if it's needed for backward compatibility, but all of our packages should be consistent. It's really weird for clients to have to use a default export for one component and a named export for another component.

Reply:
Come to think of it, I don't think any of our `index` files have default exports. The other modules do.

Reply:
Good point! I added https://github.com/material-components/material-components-web/pull/4407#discussion_r256948622 to remind myself to remove these in all `index.ts` and `component.ts` files.

Reply:
Removed `default` export from `index.ts` and `component.ts`